app/controllers/concerns/signature_verification.rb

```diff
@@ -227,7 +227,7 @@ def body_digest
   end
 
   def to_header_name(name)
-    name.split(/-/).map(&:capitalize).join('-')
+    name.split('-'.map(&:capitalize).join('-')
```

Comment:
Missing `)`

Reply:
Whoops, that's what I get for manually undoing the autofix to look at he diff, than manually typing it back
```suggestion
    name.split('-').map(&:capitalize).join('-')
```